### PR TITLE
Mobile skeleton Project for code review 

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
In this project, I have set up a "Mobile portfolio" repository.
The required tasks include.

- Creating a new branch called mobile_skeleton according to GitHub flow requirements.
- Setup linter to flag any errors or bugs.
- Added style.css page, styling my mobile repository HTML page
- Added the .gitignore file that contains the node_modules, which should be ignored by git.
- Added index.html file using flex 
- Added the MIT file to grant permissions and indemnify developers for future use.
- Added a professional README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css"/>
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+    <div class="container">
+    <div class="wrapper">
+        <header><h1>Alfred Mobile Portfolio</h1></header>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please remove the `h1` tag from the header and add it in the headline section and put inside the header the following.
- The logo (if the design has one) of the design is on the left inside a link.
- The hamburger button is on the right and has a `nav` tag.

- [ ] Please put the proper background image (if the design has one).
- [ ] The header should has has the margins (top, bottom, left, right) defined in the design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        
+          <button class="btn1 btn" type="button"> HTML/CSS</button>  
+          <button class="btn2 btn" type="button">Ruby on Rails</button>  
+          <button class="btn3 btn" type="button">JavaScript</button>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please remove these buttons it is not professional to put them like that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          <button class="btn3 btn" type="button">JavaScript</button> 
+                 
+        
+        <div class="sec"><img src="image/sc.png" alt="logo"/></div>        
+        <div class="headline"> 
+            <button class="img1 im" type="button"><img src="image/sc.png" alt="logo"/></button>  
+            <button class="img2 im" type="button"><img src="image/sc.png" alt="logo"/></button>  
+            <button class="ing3 im" type="button"><img src="image/sc.png" alt="logo"/></button>
+            <button class="img4 im" type="button"><img src="image/sc.png" alt="logo"/></button> 
+        </div>
```
Comment (ShahierNashaat) [inline index.html]: - [ ] The headline should be in a section tag.
- [ ] The section has the proper background image (decorations are always background images, not `img` tags).
- [ ] The headline should have The subtitle (if existing) uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`.
- [ ] The headline should have the list of icons that uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).
- [ ] The headline should aligned with flexbox.
- [ ] The headline should has the margins (top, bottom, left, right) defined in the design.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+            <button class="img4 im" type="button"><img src="image/sc.png" alt="logo"/></button> 
+        </div> 
+        <div class="tex">
+            <p>Hello, my name is Alfred Kinyua 
+                and I am a software 
+                developer from Kenya, with two year of experience and 
+                currently studying at Microverse. 
+                Feel free to engage with me in any web based 
+                project you might have. 
+                Thank you for visiting my website.
```
Comment (ShahierNashaat) [inline index.html]: - [ ] Please close the `p` tag and add it to the headline section.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @EasyPanther , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
Hello Sir, please look at my work and advice. I attempted a new template to better my understanding, that's what its taken two like two days.Thanks

**Comment by ShahierNashaat** [comment]
Hi @EasyPanther 
- Please make sure to put inside the header only the logo and the hamburger button.
- The logo should be inside an `a` tag.
- The hamburger button should be inside a `nav` tag.

- Also make sure to put the social icons inside the `section` tag.

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
Hello sir, I have implemented all the required changes. 
- [ok ] Please make sure to put inside the header only the logo and the hamburger button.
- [ok ] The logo should be inside an a tag.
- [ok] The hamburger button should be inside a nav tag.
- [ok] Also make sure to put the social icons inside the section tag.

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
please have a look and recommend any changes. Thanks

File: index.html [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+            <div class="social">
+                <ul>
+                    <li><a href="https://twitter.com/alf" target="_blank" rel="noopener"><img rel="noreferrer" src="images/first/twitter.png" alt="Twitter Logo" ></a></li>
+                    <li><a href="https://linkedin.com/in/alf" target="_blank" rel="noopener"><img rel="noreferrer" src="images/first/n.png" alt="Linkedin Logo" ></a></li>
+                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img rel="noreferrer" src="images/first/medium.png" alt="Medium Logo" ></a></li>
+                    <li><a href="https://github.com/EasyPanther" target="_blank" rel="noopener"><img rel="noreferrer" src="images/first/github.png" alt="GitHub Logo"></a></li>
+                    <li><a href="https://angel.co/" target="_blank" rel="noopener"><img rel="noreferrer" src="images/first/angellist.png" alt="Angellist Logo"></a></li>
+                </ul>
+            </div>
+        </section>
```
Comment (ShahierNashaat) [inline index.html]: Hey @EasyPanther Please put all of this code inside one `section` tag

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
Hello sir, I have done some updates to improve on my work, putting in place all recommendations. 
please have a look.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                            <li class="portfolio item"><a href="#">Portfolio</a></li>
+                            <li class="about item"><a href="#">about </a></li>
+                            <li class="contacts item"><a href="#">contacts</a></li>
+                            <li class="ham"><a href="#"><img src="images/first/main/Union.png" alt=""></a></li>
+
+                        </ul>
+                    </nav>
+                </div>
+            </header>
+            <div class="paragraph">
```
Comment (ShahierNashaat) [inline index.html]: Hey @EasyPanther  Start the `section` tag after the `header` tag and remove the `section` from line 15.

Comment (Alfred-KInyua) [reply index.html]: thanks for the feedback

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
Please have a look at it.

**Comment by Alfred-KInyua** [comment]
@ShahierNashaat 
Hello sir, do you have any reviews for me?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="flexcontainer">
+             <header>
+                <div class="log">
+                    <div class="logo"><img src="images/first/MyLogo.png" alt=""></div>
+                </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Kindly wrap this logo with `a` tag because it's a link so it should be in `a` tag

File: index.html [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+                               <li class="ham"><a href="#"><img src="images/first/main/Union.png" alt=""></a></li>
+
+                        </ul>
+                    </nav>
+                </div>
+            </header>
+            <section>
+            <div class="paragraph">
+                <ul>
+                    <li class="heading"><strong>I'm Alfred Kinyua Glad to see you! </strong></li>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Kindly use this in `h1` tag because it is a heading.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+                    <li class="about">
+                        I’m a software developer! I can help you build a product ,
+                        feature or website Look through some of my work and
+                        experience! If you like what you see and have a project
+                        you need coded, don’t hestiate to contact me.
+                    </li>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] I would suggest you use this paragraph in the `p` tag

File: index.html [hunk 10]
```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="johns.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Document</title>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You didn't link the HTML with CSS So it is not displaying on the browser and the design is not like as Figma kindly fix it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfred-KInyua** [comment]
@roshan-bajgain 
Hello, 
Please review the changes made. I received an error when I used p in ul li items from webhint. kindly advice

**Comment by roshan-bajgain** [comment]
@EasyPanther I just clone your project and check after changing the review made by me and it was working perfectly. Just I recommend you that remove the `ul` tag and in place of the `li` tag use `p` tag.

**Comment by Alfred-KInyua** [comment]
@roshan-bajgain 
Thanks for the review, please have a look at my project.

**Review by keroloussamy — APPROVED** [review]
Hi @EasyPanther, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**